modules/seqexec/server/src/main/scala/seqexec/server/gems/GemsControllerEpics.scala

```diff
@@ -156,9 +153,9 @@ object GemsControllerEpics {
   val Log = getLogger
 
   def apply[F[_]: Async](epicsSys: => GemsEpics[F],
-                         gsaoiSys: => GsaoiEpics[F]
+                         gsaoiGuider: => GsaoiGuider[F]
```

Comment:
I think this may be pass by value. `GmosEpics` needs to be passed by ref due to the initalization process